content/en/platform/corda/5.2/deploying-operating/deployment/upgrading521/_index.md

```diff
@@ -0,0 +1,517 @@
+---
+description: "Learn how to upgrade your cluster from Corda 5.2 to Corda 5.2.1."
+date: '2024-05-15'
+title: "Upgrading from 5.2 to 5.2.1"
+menu:
+  corda52:
+    parent: corda52-cluster-deploy
+    identifier: corda52-cluster-upgrade521
+    weight: 5000
+---
+# Upgrading from 5.2 to 5.2.1
+
+This section describes how to upgrade a Corda cluster from version 5.2 to 5.2.1.
+
+{{< note >}}
+You cannot upgrade Corda 5.1 to 5.2.1. You must upgrade Corda to version 5.2 first. For information on how to do it, see [Upgrading from 5.1 to 5.2]({{< relref "../upgrading/_index.md" >}}).
+{{< /note >}}
+
+The following section includes script excerpts which you can use as a basis for performing upgrades. The entire script is included at the end. The script is based on upgrading a cluster running on Docker Desktop on a local host using the R3-provided Corda Helm chart for installing PostgreSQL and Kafka. However, you must tailor this script for the specific environment being updated.
+
+The examples provided in this section assume that you installed Corda 5.1 in a namespace called `corda`. This is different to other deployments.
+
+To perform an upgrade, you must fulfill the required [prerequisites](#prerequisites) and go through the following steps:
+
+1. [Back Up the Corda Database](#back-up-the-corda-database)
+2. [Test the Migration](#test-the-migration)
+3. [Migrate Data From Kafka Topics to the Cluster Database](#migrate-data-from-kafka-topics-to-the-cluster-database)
+4. [Scale Down the Running Corda Worker Instances](#scale-down-the-running-corda-worker-instances)
+5. [Migrate the Corda Cluster Database](#migrate-the-corda-cluster-database)
+6. [Optional: Clear Down Key Rotation Stale Data](#optional-clear-down-key-rotation-stale-data)
+7. [Update Kafka Topics](#update-kafka-topics)
+8. [Launch the Corda 5.2.1 Workers](#launch-the-corda-521-workers)
+
+For information about how to roll back an upgrade, see [Rolling Back]({{< relref "rolling-back521.md" >}}).
+
+Following a platform upgrade, Network Operators should upgrade their networks. For more information, see [Upgrading an Application Network]({{< relref "../../../application-networks/upgrading/_index.md" >}}).
```

Comment:
This shouldn't be necessary when migrating from 5.2 to 5.2.1 and so this line should be removed.